patches/@types+marked+0.6.5.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/@types/marked/index.d.ts b/node_modules/@types/marked/index.d.ts
+index 0cbdf01..db97f38 100644
+--- a/node_modules/@types/marked/index.d.ts
++++ b/node_modules/@types/marked/index.d.ts
+@@ -90,6 +90,7 @@ declare namespace marked {
+ 
+     class Renderer {
+         constructor(options?: MarkedOptions);
++        options: MarkedOptions;
```

Comment:
It was on definitely typed repo. `@types/marked`.

Reply:
Yeah, that's totally fine. There's nothing that needs to be changed here.

Reply:
Hi @hmhealey, sorry my previous reply does not answer your question. I've overlooked it, the file in question is the patch. I've added the `options` type because [line 139 on renderer.tsx](https://github.com/karlmarxlopez/mattermost-webapp/blob/migrate-to-typescript-12481/utils/markdown/renderer.tsx#L139) is accessing `this.options` member, which does not exist on `marked.Renderer` type.